src/screen/AddPostScreen.js

```diff
@@ -28,19 +26,13 @@ const AddPostScreen = ({ setTitleAction, navigation }) => {
         <Button
           title="Add"
           onPress={() => {
-            setTitleAction(title, text) && navigation.navigate('Home');
+            sendPost({ title, text });
+            goTo('Home');
           }}
         />
       </Block>
     </Block>
   );
 };
 
-const mapStateToProps = store => ({
-  createPost: store.post.posts
-});
-const mapDispatchToProps = dispatch => ({
-  setTitleAction: (title, text) => dispatch(createPostTitle(title, text)),
-});
-
-export default connect(mapStateToProps, mapDispatchToProps)(AddPostScreen);
+export default AddPostScreen;
```

Comment:
`export default function AddPostScreen({ navigation }){...`? or with arrow. doesn't matter.

Reply:
just use one code style for all